app/Views/Templates/components/actions/dropdown.blade.php

```diff
@@ -0,0 +1,60 @@
+@props([
+    'variant' => 'regular', // Dropdown variant: regular or card
+    'contentRole' => 'primary', // Content role: primary, secondary, or tertiary
+    'position' => 'bottom', // Dropdown position: left, right, top, bottom, inner, outer
+    'label-text' => 'Dropdown', // Text for the dropdown button
+    'card-label' => 'Card Title!', // Text for the card title
+    'caption' => ''
+])
+
+@php
+    // Determine the button class based on the content role
+    $buttonClass = match($contentRole) {
+        'secondary' => 'btn btn-secondary',
+        'tertiary' => 'btn btn-accent',
+        default => 'btn btn-primary',
```

Comment:
default in this case and based on the dasiyUI page should probably just be "btn" 
then we can have primary, secondary etc.

Lets go and allow the full list of options:
primary, secondary, accent, ghost(=tertiary), link